test/functional/normal/put_spec.lua

```diff
@@ -1,19 +1,15 @@
 local helpers = require('test.functional.helpers')(after_each)
 
--- TODO Include tests for folds
-
 local clear = helpers.clear
 local insert = helpers.insert
-local eval = helpers.eval
 local feed = helpers.feed
 local expect = helpers.expect
 local eq = helpers.eq
 local execute = helpers.execute
 local curbuf_contents = helpers.curbuf_contents
 local funcs = helpers.funcs
-local curwinmeths = helpers.curwinmeths
 local dedent = helpers.dedent
-local setreg, getreg = funcs.setreg, funcs.getreg
+local getreg = funcs.getreg
 
 local function setup()
```

Comment:
Is there a reason for not using busted's `setup(function()...end)` (runs once per `describe()` in the current scope) or `before_each(function()...end)` (runs once per `it()` in the current scope) methods?